packages/plugin-core/src/_extension.ts

```diff
@@ -439,9 +439,17 @@ export async function _activate(
       previousGlobalVersion,
       currentVersion,
     });
+    // set initial install ^194e5bw7so9g
     if (extensionInstallStatus === InstallStatus.INITIAL_INSTALL) {
       MetadataService.instance().setInitialInstall();
     }
+
+    // TODO: temporary backfill
+    if (_.isUndefined(MetadataService.instance().getMeta().firstInstall)) {
+      const time = Time.DateTime.fromISO("2022-01-01");
```

Comment:
its actually useful for historical analysis to try to do a conservative estimate here. we started tracking `firstInstall` with the following commit. setting the `firstInstall` date to the release date `2021-06-22` as a conservative guess for actual `firstInstall`

commit f07820cdbefe56c00b3e123dde43ec6329b26469
Author: Kevin Lin <kevin@dendron.so>
Date:   Sat Jun 19 17:41:44 2021 -0700

    chore: add metadata service

Reply:
Can you remove 'TODO' from the comment, as the backfill looks like it's 'done' 😄 

In your comment you refer to setting it to '2021-06-22', but the code is '2022-01-01'?  I think '2021-06-22' would be more accurate? 

Another thing to be aware of is that this metadata service flag is different from the 'install' telemetry event, which goes back to `2021-04-26`

Reply:
the reason i added the TODO is because I want to remove this logic after we drive the number of folks without a `firstInstall` property to zero. there might be other cases when this value is missing (eg. user deleted the file) but that would be handled differently in the future. 

as for the dates, i updated the date to account for kaan's comment, haven't update the comment. will update.